python/TestHarness/TestHarness.py

```diff
@@ -239,6 +239,10 @@ def findAndRunTests(self, find_only=False):
                 os.chdir(saved_cwd)
                 sys.path.pop()
     except KeyboardInterrupt:
+      try:
+        self.writeFailedTest.close()
```

Comment:
Curious, did this file fail to close during your testing? If not, I'd prefer to remove this try/catch. I'd rather know why something is failing or if it ever fails first. Please remove or document why this is here.